src/constants/chains.ts

```diff
@@ -30,6 +32,7 @@ export const CHAIN_IDS_TO_NAMES = {
   [SupportedChainId.ARBITRUM_GOERLI]: 'arbitrum_goerli',
   [SupportedChainId.OPTIMISM]: 'optimism',
   [SupportedChainId.OPTIMISM_GOERLI]: 'optimism_goerli',
+  [SupportedChainId.BOBA]: 'boba',
 }
 
```

Comment:
lets add support for BOBA_GOERLI too to the interface (for the testnet)

(although, i'm not sure what the future of goerli is - but its what we have now, we could probably replace this when we change our testnet)

Reply:
Goerli has been added